extensions/kafka-connect-jdbc/LICENSE

```diff
@@ -0,0 +1,114 @@
+                    Confluent Community License Agreement
```

Comment:
Is license compatible

Reply:
we have checked with the legal team, it should be okay if are not the competitor of their commercial products